src/observe.js

```diff
@@ -1685,7 +1685,8 @@
 
   var expose = global;
 
-  if (typeof exports !== 'undefined') {
+  if (typeof exports !== 'undefined' && exports + '' === '[object Object]' &&
```

Comment:
Yes, they're equivalent. I'll do the test the other way, as it's more explicit as to what it's doing.

I'll look if other libraries have this same pattern and test that `exports` is an empty plain `Object`. Looking at the underscore.js code, it'd have this same bug. [How lodash does this test](https://github.com/lodash/lodash/blob/master/lodash.js#L265) seems more resilient. Maybe I'll use that as a basis.